clang/test/Driver/sycl-offload.cpp

```diff
@@ -0,0 +1,9 @@
+/// Check "-aux-target-cpu" and "target-cpu" are passed when compiling for SYCL offload device and host codes:
+//  RUN:  %clang -### -fsycl -c %s 2>&1 | FileCheck -check-prefix=CHECK-OFFLOAD %s
+//  CHECK-OFFLOAD: "-aux-target-cpu" "x86-64"
```

Comment:
Let's make it a little bit more reliable:
```suggestion
//  CHECK-OFFLOAD: clang
//  CHECK-OFFLOAD-SAME: "-fsycl-is-device"
//  CHECK-OFFLOAD-SAME: "-aux-target-cpu" "x86-64"
```

Reply:
Thanks @bader for the suggestion. I have updated the test.